package.json

```diff
@@ -36,11 +36,11 @@
     "grunt-contrib-jshint": "^3.1.1",
     "grunt-contrib-nodeunit": "^4.0.0",
     "load-grunt-tasks": "^5.1.0",
-    "postcss": "^8.3.9",
-    "postcss-scss": "^4.0.1"
+    "postcss": "^8.3.11",
+    "postcss-scss": "^4.0.2"
   },
   "peerDependencies": {
     "grunt": ">=1.0.4",
-    "postcss": "^8.2.9"
+    "postcss": "^8.0.0"
```

Comment:
I changed this to 8.0.0 because that version should also work. So, there's no need to update the peer dependency too.